app/uk/gov/hmrc/cataloguefrontend/connector/UserManagementConnector.scala

```diff
@@ -145,11 +124,10 @@ trait UserManagementConnector extends UserManagementPortalLink {
   }
 
 
-
-
   def getTeamDetails(team: String)(implicit hc: HeaderCarrier): Future[Either[UMPError, TeamDetails]] = {
+    import uk.gov.hmrc.cataloguefrontend.UserManagementConnector.teamDetailsReads
```

Comment:
Implicits in the companion object imported by default in the implicit scope. You can remove this.